web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -682,6 +784,36 @@ const TeamReviews = ({ periodId }) => {
         onSelect={handleNewRequest}
         onClose={handleCloseNewRequest}
       />
+      <MemberSelectorDialog
```

Comment:
This is used to display the team members associated with this review period.
It also allows more to be added and existing selections to be removed.